docs/learn/learn-xcm.md

```diff
@@ -0,0 +1,299 @@
+   the sender with an additional message.
+4. **Agnostic**: XCM makes no assumptions about the nature of the Consensus Systems between which the 
+   messages are being passed.
+
+:::note 
+
+XCM is a work-in-progress. XCM v2 is deployed on Polkadot and v3 is currently in
+development. See Gavin's talk about v3 in the [resources](#resources) section.
+
+:::
+
+## A Format, Not a Protocol
+
+What started as an approach to _cross-chain communication_, has evolved into a format for
+[**Cross-Consensus Communication**](https://polkadot.network/cross-chain-communication) that is
+not only conducted between chains, but also between smart contracts, pallets, bridges, and even
+sharded enclaves like [SPREE](learn-spree.md).
+
+XCM cannot actually send messages between systems. It is a format for how message transfer should be
+performed, similar to how RESTful services use REST as an architectural style of development.
+
+Similar to UDP, out of the box XCM is a "fire and forget" model, unless there is a seperate XCM message
+designed to be a response message which can be sent from the recipient to the sender. Any kind of error
+handling should also be done on the recipient side.
+
+:::info
+
+XCM is not designed in a way where every system supporting the format is expected to be able to interpret
+any possible XCM message. Practically speaking, one can imagine that some messages will not have
+reasonable interpretations under some systems or will be intentionally unsupported.
+
+:::
+
+### Example Use-Cases
+
+- Request for specific operations to occur on the recipient system such as governance voting.
+- Enables single use-case chains e.g. [Statemint/e](/learn/learn-statemint.md) as asset parachains
+- Optionally include payment of fees on a target network for requested operation.
+- Provide methods for various asset transfer models:
+  - **Remote Transfers**: control an account on a remote chain, allowing the local chain to have an
+    address on the remote chain for receiving funds and to eventually transfer those funds it
+    controls into other accounts on that remote chain.
+  - **Asset Teleportation**: movement of an asset happens by destroying it on one side and creating a clone
+    on the other side.
+  - **Reserve Asset Transfer**: there may be two chains that want to nominate a third chain, where
+    one includes a native asset that can be used as a reserve for that asset. Then, the derivative
+    form of the asset on each of those chains would be fully backed, allowing the derivative asset
+    to be exchanged for the underlying asset on the reserve chain backing it.
+
+Let's review two of these example asset transfer use cases: **Asset Teleportation** and **Reserve Asset Transfer**.
+
+### Asset Teleportation
+An asset teleport operation from a single source to a single destination. 
+
+![](../assets/cross-consensus/xcm-asset-teleportation.png)
```

Comment:
If SEO is important, consider adding a description for this image, something like:

```suggestion
![Diagram of the usage flow while teleporting assets](../assets/cross-consensus/xcm-asset-teleportation.png)
```